kms/templates/enable_iam.json.tpl

```diff
@@ -10,6 +10,44 @@
       },
       "Action": "kms:*",
       "Resource": "*"
+    },
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "sns.amazonaws.com"
+      },
```

Comment:
This section of the policy allows the SNS topic to pass messages to the encrypted SQS queues. The only queues that are triggered from the SNS topic are the download-files and failure queues so these are included in the condition.